var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -85,17 +93,46 @@ def languages(self):
 
     def configure_args(self):
         spec = self.spec
-        cc = self.compiler.cc
-        cxx = self.compiler.cxx
+
+        if '+intel_backend' in spec:
+            cc = which('mpicc')
+            cxx = which('mpic++')
```

Comment:
Does your "already existing license file" look correct?

`intel-parallel-studio` comes with both an Intel compiler and Intel MPI. But not all Intel installations come with both.